.nais/nais.yaml

```
@@ -70,5 +70,11 @@ spec:
     sidecar:
       enabled: true
       level: Level4
+      autoLogin: true
+       autoLoginIgnorePaths:
+        - "{{base_path}}/api/*"
+        - "{{base_path}}/api/*/*"
+        - "{{base_path}}/api/*/*/*"
+        - "{{base_path}}/_next/static/*/*"
```

```suggestion
       autoLoginIgnorePaths:
        - "{{basePath}}/api/*"
        - "{{basePath}}/api/*/*"
        - "{{basePath}}/api/*/*/*"
        - "{{basePath}}/_next/static/*/*"
```

I denne appen ser det ut som det er `basePath` som brukes.